device/transport/mqtt/src/mqtt_receiver.ts

```diff
@@ -207,15 +188,10 @@ export class MqttReceiver extends EventEmitter implements Receiver {
                         self._topics.message :
                         null;
       if (!!topic) {
+        debug('removing listener for topic: ' + JSON.stringify(topic));
         // decrement listeners count for this topic
         --(topic.listenersCount);
```

Comment:
>) [](start = 31, length = 1)

Did you REALLY want this in the parens?  Are you decrementing the value as opposed to the object?